addon/components/base/bs-alert.js

```diff
@@ -16,9 +16,9 @@ import listenTo from 'ember-bootstrap/utils/listen-to-cp';
  `type` property for proper styling.
 
  ```hbs
- {{#bs-alert type="success"}}
- <strong>Well done!</strong> You successfully read this important alert message.
- {{/bs-alert}}
+ <BsAlert @type="success">
+  <strong>Well done!</strong> You successfully read this important alert message.
```

Comment:
It should be two spaces to be inline with default template linting rules, shouldn't it?

Reply:
I tried to indent everything correctly, but I'm unsure, to be honest, would like a review here.